Extension scrollable consent form description (#2)
ramseyTA/android-consent-sdk: README.md

 
 Object `consentFormData` provides all needed data for displaying consent form.

### Consent form display options

> **Optionally** you can customize the display options for you consent form dialog/dialogfragment/fragment/activity by passing a `ConsentFormDisplayOptions` object in your showConsentForm function call.<br><br>

Reviewer: ```suggestion
**Optionally** you can customize the display options for your consent form dialog/dialogfragment/fragment/activity by passing a `ConsentFormDisplayOptions` object into your showConsentForm function call.<br><br>
```

> Until now there is only the option to decide by setting the `consentFormDescriptionScrollingOnly` flag, if you want the whole layout scrollable or only title & description.<br>

Reviewer: ```suggestion
With the `consentFormDescriptionScrollingOnly` flag, you can make the whole layout scrollable or only title & description.<br>
```

Until now? Will this change otherwise mybe just the current state
